docs/migration/v8-to-v9.md

```diff
@@ -53,6 +53,8 @@ In preparation for the OpenTelemetry SDK v2, which will raise the minimum requir
 
 Additionally, like the OpenTelemetry SDK, the Sentry JavaScript SDK will follow [DefinitelyType's version support policy](https://github.com/DefinitelyTyped/DefinitelyTyped#support-window) which has a support time frame of 2 years for any released version of TypeScript.
 
+Older Typescript versions _may_ still work, but we will not test them anymore and no more guarantees apply.
```

Comment:
Added this note, not 100% sure if this makes sense? 🤔

Reply:
yeah that feels reasonable to me.

Reply:
No hard opinions but personally if a sentence effectively says nothing, which is the case for this one I believe, I would not write it down.